docs/sources/helm-charts/mimir-distributed/migration-guides/migrate-from-single-zone-with-helm.md

```diff
@@ -33,7 +35,7 @@ Depending on what version of the `mimir-distributed` Helm chart is installed cur
         enabled: false
       ```
 
-      > **Note**: a direct upgrade from non-zone aware ingesters to zone-aware ingesters will cause data loss othewise.
+      > **Note**: a direct upgrade from non-zone aware ingesters to zone-aware ingesters will cause data loss otherwise.
```

Comment:
We have recently changed the admonition style; please see https://grafana.com/docs/writers-toolkit/write/shortcodes/#admonition-shortcode for how to use the new syntax.